docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -139,6 +139,35 @@ EKS Anywhere will add by default.
 Modifying the labels associated with the control plane configuration will cause new nodes to be rolled out, replacing
 the existing nodes.
 
+#### controlPlaneConfiguration.upgradeRolloutStrategy
+Configuration parameters for upgrade strategy.
+
+#### controlPlaneConfiguration.upgradeRolloutStrategy.type
+Default: `RollingUpdate`
+
+Type of rollout strategy. Supported values: `RollingUpdate`,`InPlace`.
+
+#### controlPlaneConfiguration.upgradeRolloutStrategy.rollingUpdate
+Configuration parameters for customizing rolling upgrade behavior.
```

Comment:
can you add a line here saying this can only be configured if `upgradeRolloutStrategy.type` is `RollingUpdate`